update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -47,8 +47,7 @@ public static void main( String[] args ) throws IOException {
       String hostGK = props.getProperty("curator.database.host");
       String databaseTR = props.getProperty("release_current.name");
       String databaseGK = props.getProperty("curator.database.name");
-      authorIdTR = Integer.valueOf(props.getProperty("personId"));
-      authorIdGK = Integer.valueOf(props.getProperty("personId"));
+      personId = Integer.valueOf(props.getProperty("personId"));
       int portTR = Integer.valueOf(props.getProperty("release.database.port"));
       int portGK = Integer.valueOf(props.getProperty("curator.database.port"));
```

Comment:
It's preferable to use `Integer.parseInt(...)` for these lines since the values are being stored as primitive `int` values:

https://stackoverflow.com/a/7355056/2295778